pkg/kev/config/defaults.go

```diff
@@ -24,7 +24,7 @@ const (
 	DefaultVolumeSize = "100Mi"
 
 	// DefaultVolumeStorageClass default PV storage class
-	DefaultVolumeStorageClass = ""
+	DefaultVolumeStorageClass = "standard"
```

Comment:
We shouldn't make that assumption. Let's leave the default as an empty string as previously.

Reply:
Sure. If it's empty, then don't actually need the `DefaultVolumeStorageClass` const at all.